build/yaml/templates/npm-package-steps.yml

```diff
@@ -4,14 +4,31 @@ steps:
   inputs:
     versionSpec: '14.x'
 
+- powershell: |
+    # Get existing version from package.json
+    $packageJson = Get-Content package.json;
+    $packageJsonData = $packageJson | ConvertFrom-Json;
+
+    "VERSION: $($packageJsonData.version)"
+    "SUFFIX: $(VersionSuffix)";
+    $version = $($packageJsonData.version) + "-" + "$(VersionSuffix)";
+
+    # Replace matched version
+    (Get-Content package.json) `
```

Comment:
Why not just update $packageJsonData.version directly, then ConvertTo-Json | Out-File? Seems a little simpler than regex matching.

Reply:
Sure that would be simpler but that's not exactly what one thinks of working after 10PM :P

Reply:
I mean, the real question is why you're working after 10:00pm, but I'll allow it :P lol